src/character_clothing.cpp

```diff
@@ -0,0 +1,348 @@
+#include "character.h"
```

Comment:
Good point. I hesitated to include the armor stuff because of the name I chose. Sticking with nouns, maybe `character_attire.cpp` would be a more... "suit"-able name?

Reply:
that happens to be relevant to what i'm doing for clothing, though i have to take a 1 week hiatus. thought i'd at least mention its relevance before i disappear for a bit

Reply:
Would "character_equipment" indicate an even broader category of everything that can be worn than "character_attire", or would it indicate things wielded as well? "character_wearables" would definitely stress the "wear" aspect. That is, of course, assuming you actually want things like guns on straps to be included.

Reply:
"character_equipment" feels a little too broad to me, at least for this initial PR. "character_wearables" is a decent name, but I honestly prefer the more concrete noun "attire". There's probably going to be some ambiguity and grey area no matter what name we use here, but "things worn on the body" is indeed the concept I'm going for, and I think "attire" captures that well.